packages/driver/src/cy/commands/navigation.ts

```diff
@@ -90,7 +90,7 @@ const timedOutWaitingForPageLoad = (ms, log) => {
   }
 }
 
-const cannotVisitDifferentOrigin = ({ remote, existing, previousUrlVisited, log, isMultiDomain = false }) => {
+const cannotVisitDifferentOrigin = ({ remote, existing, previousUrlVisited, log, isCrossOriginSpecBridge = false }) => {
```

Comment:
Okay - again getting bearings 😬 

Did you say we establish a spec bridge per orgin right? Not per command  or per spec necessarily?

I'm just wondering why `isCrossOriginSpecBridge` instead of `isCrossOriginBridge`

Reply:
so this connection persists between specs ?

Reply:
yes it does

Reply:
gotttchya, then this makes sense - think I'm catching up with the basics - thanks!